### ISSUE TITLE
UI: In-game

### ISSUE THREAD

**Issue by Lithrun** [AoTTG-2/AoTTG-2#19]
**Is your feature request related to a problem? Please describe.**
Develop a new user interface for in-game. "In-game" refers to: the moment you are connected into a lobby.

@AoTTG-2/art-team 
- [x] Develop draft for how main in-game UI should look like
- [x] Character selection menu draft (Before you spawn, you first have to select a character / loadout.)

@AoTTG-2/development-team 
Art team design: https://cdn.discordapp.com/attachments/585552221425631254/688887243846582376/AoTTG_2_UI_Blades.png
Avoid hardcoding! Make use of unity UI.

- [x] Add hook & speed indication (Middle of screen. Follows mouse.)
- [x] Add Titan hit damage output
- [x] Gas & weapon ammo left (blades, AHSS rounds or thunder spears)
- [x] Player names.
- [x] Titan health

The following UI components should eventually be allowed to be moved by players, so keep this in mind:
- [x] Minimap
- [x] Chat
- [x] Playerlist
- [x] Spawn Menu (Select Human, Equipment Set and PT. Regular character sets will be done in new issue once #33 is done.)

**Comment by Lithrun** [comment]
In-game settings management is another issue.

**Comment by Lithrun** [comment]
- Player skill selection bar (top right. Limit this to one skill for now, as currently players can only have one skill, whereas later on they can have multiple)
- Combo meter (Bottom, right. Make this a mock, but it should be filled with red, where 0 = empty and 100 = full). Actual implementation details will be moved to another issue)
- Items. (Bottom left.) Add components for this. Actual implementation will be done in another issue.

These UI components depend on other issues and cannot be implemented yet. Thereby they are reserved for other issues.

**Comment by Lithrun** [comment]
Game room info reserved for another issue, depending on outcome of: https://discordapp.com/channels/681641241125060652/683548782155005963/694651619576643614
